src/App.tsx

```diff
@@ -1,14 +1,46 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[] | null>(null);
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+  const [completedStatus, setCompletedStatus] = useState('all');
```

Comment:
Create enum or union type for it
```suggestion
  const [completedStatus, setCompletedStatus] = useState('all');
```
```suggestion
  const [completedStatus, setCompletedStatus] = useState<Status>('all');
```